src/promptflow/tests/sdk_cli_azure_test/recording_utilities/constants.py

```diff
@@ -2,8 +2,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-TEST_RUN_LIVE = "PROMPT_FLOW_TEST_RUN_LIVE"
-SKIP_LIVE_RECORDING = "PROMPT_FLOW_SKIP_LIVE_RECORDING"
+ENVIRON_TEST_MODE = "PROMPT_FLOW_TEST_MODE"
+
+
+class TestMode:
+    LIVE = "live"
+    RECORD = "record"
+    REPLAY = "replay"
```

Comment:
actually you can use any value in env vars.
please review all the places of not is_live() is_record() or is_replay()

Reply:
yes, these are the expected values, and we will not recognize others;
also, have checked & updated those references.